packages/flutter/test/widgets/spell_check_test.dart

```diff
@@ -338,4 +338,80 @@ void main() {
 
     expect(textSpanTree, equals(expectedTextSpanTree));
   });
+
+  test(
+      'findSuggestionSpanAtCursorIndex finds correct span with cursor in middle of a word',
```

Comment:
These tests for findSuggestionSpanAtCursorIndex probably should be moved to editable_text_test.dart now.